src/filesystem/impls/filer/FilerFileSystem.js

```diff
@@ -173,7 +175,6 @@ define(function (require, exports, module) {
             stat(path, callback);
         });
     }
-
```

Comment:
Add this blank line back again, please.  It's a good idea not to make changes in a file that aren't related to what you're working on, so the the `git blame` info stays consistent.